doc/TRAINING.rst

```diff
@@ -195,6 +195,13 @@ Producing a mmap-able model is as simple as:
 
 Upon sucessfull run, it should report about conversion of a non-zero number of nodes. If it reports converting ``0`` nodes, something is wrong: make sure your model is a frozen one, and that you have not applied any incompatible changes (this includes ``quantize_weights``\ ).
 
+If this command returns ``convert_graphdef_memmapped_format: command not found`` it is probably because this binary is not in the ``PATH``. In order to solve this:
+
+.. code-block:: bash
+
+   $ chmod +x convert_graphdef_memmapped_format
+   $ ./convert_graphdef_memmapped_format --in_graph=output_graph.pb --out_graph=output_graph.pbmm
```

Comment:
Thanks, but I think this should rather be done by  `util/taskcluster.py`. Any opinion @reuben ?

Reply:
Agree.

Reply:
Should I? I'm not really sure where to include it.

Reply:
@juandspy I guess around here https://github.com/mozilla/DeepSpeech/blob/master/util/taskcluster.py#L147-L150 after the download, you could just force-set `+x` when this is running on some Linux/Unix system. It's not going to do big harm if it's done else than a binary, and it's going to address more transparently that issue for people not at ease enough with unix.

Reply:
Something like this?

```python
    maybe_download_tc(target_dir=args.target, tc_url=get_tc_url(args.arch, args.artifact, args.branch))

    if args.artifact == "convert_graphdef_memmapped_format":
        subprocess.check_call(['chmod', '+x', os.path.join(args.target, args.artifact)])
    
    if '.tar.' in args.artifact:
        subprocess.check_call(['tar', 'xvf', os.path.join(args.target, args.artifact), '-C', args.target])
```

Reply:
I could accept that :)

Reply:
@juandspy Now you need to remove the doc part.